apps/judicial-system/web/src/utils/hooks/useSort/useSortAppealCases.ts

```diff
@@ -0,0 +1,58 @@
+import { AppealedCasesQueryResponse } from '@island.is/judicial-system-web/src/routes/CourtOfAppeal/Cases/Cases'
+import { useState, useMemo } from 'react'
+
+const useSortAppealCases = (
+  defaultColumn: string,
+  defaultDirection: 'ascending' | 'descending',
+  data: AppealedCasesQueryResponse[],
```

Comment:
I wonder about the abstraction of the AppealedCasesQueryResponse[] vs CaseListEntry[], they're almost exactly the same, and it would be nice to use the same sort hook for all tables. But that's another refactor and another conversation so I think we should maybe just keep that for another PR 😊